### PR TITLE
Feature/user accounts tabs

### PR DESCRIPTION
**Pull request by tgreyuk**

### CODE REVIEW

File: src/app/core/services/user.service.ts [hunk 1]
```diff
@@ -13,6 +15,9 @@ export class UserService {
   private _userDetails$: BehaviorSubject<UserDetails> = new BehaviorSubject<UserDetails>(null);
   public userDetails$: Observable<UserDetails> = this._userDetails$.asObservable();
 
+  private _returnUrl$: BehaviorSubject<URLStructure> = new BehaviorSubject<URLStructure>(null);
```
Comment (ChazUK) [inline src/app/core/services/user.service.ts]: Seeing this popping up in more and more places, maybe it'd be a good idea to have it directly in the BackService?

What do you think? We can leave it as tech debt for now

File: src/app/features/create-account/create-account-routing.module.ts [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
-import { NgModule } from '@angular/core';
-import { RouterModule, Routes } from '@angular/router';
 import { SecurityQuestionComponent } from '@features/create-account/security-question/security-question.component';
 
 const routes: Routes = [
   {
     path: '',
+  },
+  {
+    path: ':establishmentUid/create-user',
```
Comment (ChazUK) [inline src/app/features/create-account/create-account-routing.module.ts]: I think we need to look at this routing, unless we are lazy loading it in the Workplace Routing

File: src/app/features/workplace/view-workplace/view-workplace.component.html [hunk 3]
```diff
@@ -14,6 +14,10 @@ <h1 class="govuk-heading-l">
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
     <app-tabs>
+      <app-tab [title]="'User accounts'">
```
Comment (ChazUK) [inline src/app/features/workplace/view-workplace/view-workplace.component.html]: I think on the View Workplace the User Accounts tab comes after the Workplace tab

File: src/app/core/services/user.service.ts (hunk 1 above)
Comment (tgreyuk) [reply src/app/core/services/user.service.ts]: Yeah maybe I haven't really looked in back service.

File: src/app/features/create-account/create-account-routing.module.ts (hunk 2 above)
Comment (tgreyuk) [reply src/app/features/create-account/create-account-routing.module.ts]: create account is its own lazy loaded module. Let me know if you want me to do anything on this?

File: src/app/features/workplace/view-workplace/view-workplace.component.html (hunk 3 above)
Comment (tgreyuk) [reply src/app/features/workplace/view-workplace/view-workplace.component.html]: Not sure but its the first tab on the screenshots in the card.

File: src/app/features/create-account/create-account-routing.module.ts (hunk 2 above)
Comment (ChazUK) [reply src/app/features/create-account/create-account-routing.module.ts]: Just spoke with @omarCreativeDev  and he's moving the Create Account stuff to within the Workplace Module

**Comment by tgreyuk** [comment]
[(6) User Accounts - Workplace Dashboard - User Accounts Tab](https://trello.com/c/SB6ppNdn/143-6-user-accounts-workplace-dashboard-user-accounts-tab)

**Comment by tgreyuk** [comment]
[(6) User Accounts - Workplace Dashboard - User Accounts Tab](https://trello.com/c/SB6ppNdn/143-6-user-accounts-workplace-dashboard-user-accounts-tab)

File: src/app/features/create-account/create-account-routing.module.ts (hunk 2 above)
Comment (tgreyuk) [reply src/app/features/create-account/create-account-routing.module.ts]: ok @omarCreativeDev note the establishmentUid param needs to be on the create page..

Comment (omarCreativeDev) [reply src/app/features/create-account/create-account-routing.module.ts]: yup am moving the create component/route to the workplace module as we speak

**Pull request merged**